samples/guestbook/README.md

```diff
@@ -185,6 +188,77 @@ directory using the same variables you entered during `terraform apply`.
 [AWS T&C]: https://aws.amazon.com/marketplace/pp?sku=55q52qvgjfpdj2fpfy9mb1lo4
 [GitHub SSH]: https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/
 
+## Running on Azure
+
+If you want to run this sample on Azure, you first need to set up an Azure
+account. Use the `az` CLI to log in.
+
+```shell
+az login
+```
+
+The Go CDK doesn't have support for SQL on Azure yet
+(https://github.com/google/go-cloud/issues/1305), so we'll run MySQL and the
+guestbook binary locally. Guestbook will get the Gopher logo and MOTD from Azure
+storage.
+
+### Provision resources with Terraform
+
+We'll use Terraform, a tool for initializing cloud resources, to set up your
+project.
+
+```shell
+# Enter the Azure directory from samples/guestbook.
+cd azure
+terraform init
+
+# Provisioning can take up to 10 minutes.
+# Keep track of the output of this command as it is needed later.
+terraform apply -var location="West US"
+
+<snip>
+Outputs:
+
+access_key = [redacted]
+storage_account = [redacated]
+storage_container = [redacted]
+```
+
+### Running
+
+You will need to run a local MySQL database server, similar to what we did for
+running locally earlier.
+
+```shell
+go get ./localdb/... # Get package dependencies.
+go run localdb/main.go
+```
+
+In another terminal, add your Azure credentials to the environment and run the
+`guestbook` application:
+
+```shell
+# Enter the storage_account from the Terraform output earlier.
+AZURE_STORAGE_ACCOUNT=<your storage_account>
```

Comment:
add `export` to the beginning of these lines?

Reply:
Done.